### PR TITLE
Credits

### PR DESCRIPTION
**Pull request by ineilson**
Remove reference to EGI-Engage and add EOSC-hub logo. Mods to layout, logos and wording on landing and left menu credits box. Add alt text to logo images for #117.

### CODE REVIEW

**Comment by tofu-rocketry** [comment]
This is dependent on #124 I presume.

**Comment by gregcorbett** [comment]
You could use the STFC logo with a transparent background rather than the one with the white background.

**Comment by gregcorbett** [comment]
Seems like we are happy with this one, but are waiting on #124

**Comment by tofu-rocketry** [comment]
Would be good to see the simplified diff once #124 is merged.

**Comment by gregcorbett** [comment]
Does this show that?: https://github.com/ineilson/gocdb/compare/privacy-updates...ineilson:credits

**Comment by tofu-rocketry** [comment]
Seems to. It's a little clearer with the whitespace ignored: https://github.com/ineilson/gocdb/compare/privacy-updates...ineilson:credits?w=1

File: htdocs/landing/index.html [hunk 1]
```diff
@@ -1,55 +1,41 @@
...
         </div>
       </div>
     </div>
-   </body>
-</html>
-  
-  
+  </div>
+</body>
+</html>
```
Comment (tofu-rocketry) [inline htdocs/landing/index.html]: Should keep a newline at the end.

Comment (tofu-rocketry) [reply htdocs/landing/index.html]:
```suggestion
</html>

```

**Comment by gregcorbett** [comment]
@ineilson, could you accept @tofu-rocketry's suggested change? I can't seem to do it myself (maybe because I didn't open the PR...)

**Pull request merged**